frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -234,7 +234,7 @@ const PlayersList = (props: {
   );
 };
 const PlayerNotFound = (props: {
-  clearInputAndAddPlayer: (name: string, surname: string) => void;
+  clearInputAddAndRegisterPlayer: (name: string, surname: string) => void;
```

Comment:
clearInput**Add**AndRegisterPlayer?

Reply:
no bo tworzy się przy okazji profil zawodnika, który idzie do ogólne puli profili

Reply:
czy usunąć to Add jako, że to jest wynik uboczny RegisterPlayer ?

Reply:
A dobra - znowu ślepo poprawiłeś jak napisałem :P 
Powinno być CreatePlayerProfileForm:
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/blob/develop/backend/src/modules/player-profiles/core/application/command/CreatePlayerProfile.ts

Bo tak się nazywa command w naszym słownictwie (zamiast Add) i wtedy chyba będzie spoko. A do turnieju Register jest OK.

Reply:
to są i tak te callbacki i funkcji o "tych samych" nazwach to tam są ze 3 lub 4 :-P 
CreatePlayerProfileForm <--- nazwa pliku i tym samym funkcji, która renderuje formularz 
a reszta to już callbacki, które idą od tego formularza w górę drzewa do TournamentRegistrations

Dobra jak pushnę zmiany co teraz robię to wrócę i tutaj pozmieniam